apps/admin_app/assets/css/promotions.css

```diff
@@ -0,0 +1,169 @@
+.App {
+    text-align: center;
+  }
+  
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+    height: 40vmin;
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
+  
+  .App-link {
+    color: #61dafb;
+  }
+  
+  .form .formrow{
+    text-align: left;
+    margin:25px auto;
+    max-width: 400px;
+  }
+  
+
+
+  .topnav{
+    background-color: rgba(255,255,255,1);
```

Comment:
@SakinaBoriwala21 as suggested by @VivekNayyar may I recommend you to fetch the background-color from a file where they are all defined in one place?

Reply:
Yes, For now I will use the standard html colors. I can just use "white" here instead of rgba(255,255,255,1)